web/packages/core/src/ruffle-player.ts

```diff
@@ -149,6 +152,8 @@ export class RufflePlayer extends HTMLElement {
     private _cachedDebugInfo: string | null = null;
 
     private isExtension = false;
+    private longPressTimer: ReturnType<typeof setTimeout> | null = null;
+    private longPressTimeout = 800;
```

Comment:
I think it's more appropriate to define this as `const` (even inside the body of `startLongPressTimer()` itself).